modules/elfutils/0.191/patches/add_build_file.patch

```diff
@@ -0,0 +1,31 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,26 @@
++# Source: https://github.com/Juniper/ebplane/blob/master/third_party/BUILD.elf
++load("@rules_foreign_cc//tools/build_defs:configure.bzl", "configure_make")
++
++licenses(["notice"])  # The libraries and backends are dual GPLv2+/LGPLv3+. The utilities are GPLv3+.
++
++filegroup(
++    name = "elfutils_all",
++    srcs = glob(["**"]),
++    visibility = ["//visibility:public"],
++)
++
++configure_make(
++    name = "libelf",
++    # elfutils can only be built with gcc. What you see below is a hack:
++    # it overrides the environment variables set by bazel to use clang with
++    # values that force detection and use of the host installed gcc.
++    # TODO: fix the build to be more hermetic.
```

Comment:
I think that this should be fixed before adding the module to the registry. Folks may just `bazel_dep` it assuming it's "Bazelized" when it's actually very non-hermetic. 

Why is gcc needed to build this? Is there something we could patch? Alternatively, you could document that this requires gcc and use a `select` or `target_compatible_with` on https://github.com/bazelbuild/rules_cc/blob/main/cc/compiler/BUILD to fail with any other compiler.